### DISCUSSION TITLE
share "__diesel_schema_migrations"

### DISCUSSION THREAD

**Discussion by nicraMarcin** [diesel-rs/diesel#2701] [Q&A]
Hi,
I'm going to write a simple app with microservices. All microservices will share the same database but should have seperate schema migratios table. 
For example:
 - users should have "__users_diesel_schema_migrations"
 - posts should have "__posts_diesel_schema_migrations"

How can I to prefix schema migrations table?

**Comment by weiznich** [comment]
This is nothing that is supported by diesel. Additionally we do not plan to support this. That written #2657 should allow you to roll out your own migration harness to configure in whatever way you want.

**Reply by nicraMarcin** [reply]
Thank you for reply. As I understand it will be available in next release?

**Reply by weiznich** [reply]
The linked PR will be part of the next release.
